script.js

```diff
@@ -0,0 +1,164 @@
+
+class RenderManager {
+    list = new ListManager();
```

Comment:
let's add private modifier to the list, e.g:
private list = ...

Reply:
great
in js the "private" is #

Reply:
amazing